place-helper.js

```diff
@@ -0,0 +1,38 @@
+var placesCall = function(place) {
+  var key = 'AIzaSyCHsQMx-gpiPsKxiKd9hhtEdR_GagDRHuw';
+  var url = `https://maps.googleapis.com/maps/api/place/textsearch/json?query=${place}&key=${key}`;
```

Comment:
we shouldnt leave our key in this file, it should be in a gitignored file and imported in, right?